Core1Platform/AdministratingCore1/02_ManageUsers.md

```diff
@@ -0,0 +1,260 @@
+> [INFO] As a user without admin rights, you can change some data too. For detailed information, see [Edit profile settings](../UsingCore1/02_EditProfileSettings.md "Edit profile settings"). 
+
+
+
+## Create user
+Users are created in the main account only.
+
+#### Prerequisites
+
+- You are logged in to a main account
+- You are granted to create users.
+
+#### Procedure
+
+*Settings > Users and groups > Tab BENUTZERVERWALTUNG (USER MANAGEMENT)*
+
+![User management](../../Assets/Screenshots/Settings/UsersGroups/UserManagement/UserManagement.png "[User management]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The screen is grayed out. A small menu is opened in the bottom right corner.
+
+   ![User management menu](../../Assets/Screenshots/Core1Platform/AdministratingCore1/UserAdd.png "[User management menu]")
+
+2. Click the ![Create user](../../Assets/Icons/Edit02.png "[Create user]") (Create user) button.   
+  The *Create user* window is opened.    
+    
+    ![Create user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/UserCreate.png "[Create user]")
+
+3. Click the *Select active option* drop-down list to select whether the new user will be able to log in or not. The following options are available: 
+    - **Active, can log in**
+    - **Inactive, user is locked**
+
+4. Click the *Password type* drop-down list to select how the password is to be created and submitted to the user. The following options are available:
+    - **Generate password and send via email**  
+      A password is generated automatically, and it is sent by Actindo to the email address specified below. 
+    - **Enter password and send by email**    
+      You define an individual password, and it is sent by Actindo to the email address specified below. 
+    - **Specify and save password**   
+      You define an individual password and save it. It is not sent via email.  
+    - **Do not change password**   
+      Use this option if you want to change the user's data, but you do not want to edit the password. 
+
+5. If you have not selected the *Generate password and send via email* option in the *Password type option* drop-down list, define the password in the *Set password* field.
+
+6. Enter the username in the *Login* field. This is the name with which the user must log in.  
+
+7. Enter the name and address of the user. Use the *Address* field to enter street and house number.
+
+8. Enter the email address of the new user.   
+  After you have saved the new user, it will be used to send the password to the user.
+
+9. If desired, click the *Role in company* drop-down list. The following options are available:
+    - Entwicklung (Development)
+    - Buchhaltung (Accounting)
+    - Logistik (Logistics)
+    - Artikelanlage (Creating products)
+    - Einkauf (Procurement)
+    - Marktplätze (Market places)
+    - Anderes (Others)
+
```

Comment:
Wir haben immer die UI-Elemente auf Englisch geschrieben, auch wenn noch nicht übersetzt waren/sind. Ich denke, es sinnvoller, das hier zu tun, damit wir konsistent bleiben. Hoffentlich werden wir diese Texte auch bald übersetzen.

Reply:
Da in diesem Fall die EN Übersetzungen nicht eindeutig sind, möchte ich erst abwarten